var/spack/repos/builtin/packages/openpa/package.py

```diff
@@ -10,7 +10,7 @@ class Openpa(AutotoolsPackage):
     (and related constructs) for high performance, concurrent software"""
 
     homepage = "https://github.com/pmodels/openpa"
-    url = "https://github.com/pmodels/openpa/releases/download/v1.0.4/openpa-1.0.4.tar.gz"
+    url = "https://github.com/pmodels/openpa/releases/download/v1.0.5/openpa-1.0.5.tar.gz"
```

Comment:
This change is not necessary.  That version already exists on line 17.  And Spack automatically performs version substitution to URLs.

Reply:
See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.